src/main/resources/plugin.yaml

```
@@ -0,0 +1,22 @@
+# Refer https://docs.halo.run/developer-guide/plugin/basics/manifest
+
+apiVersion: plugin.halo.run/v1alpha1
+kind: Plugin
+metadata:
+  # The name defines how the plugin is invoked, A unique name
+  name: plugin-sticker
```

```suggestion
  name: sticker
```

kind 和 apiVersion 已经可以用于区分是 Plugin 资源，所以没有必要在添加 plugin- 前缀。